bats/os_helper.bash

```diff
@@ -122,3 +124,23 @@ tCommandExists() {
 tFileExists() {
   [[ -f "$1" ]]
 }
+
+tRHSubscribeAttach() {
```

Comment:
Nowhere yet, I just copied os_helper.bash from foreman-bats which includes this.